api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -86,6 +87,13 @@ type KogitoServerlessPlatformSpec struct {
 	BuildPlatform api.PlatformBuildSpec `json:"platform,omitempty"`
 	// Configuration list of configuration properties to be attached to all the Workflow built from this Platform
 	Configuration ConfigurationSpec `json:"configuration,omitempty"`
+	// Trait is list of trait properties attached to all Workflows built with Platform
```

Comment:
@davidesalerno  @amadhusu the purpose of the traits is to identify which trait needs to be configured automatically, right?

if so, don't we need to define it on the cr?

@amadhusu can you please provide a example CR as well the tests for this? If the service trait is auto == true, the service for the workflows needs to be created.

Reply:
@spolti There is a sample CR with a basic test case for checking the workflow. I will work on the controller for the implementation of this so we can add more test cases and validate the same. I will create other examples and test cases for different Service Types as well once I am done with the Controller Implementation for the same.